README.md

```diff
@@ -5,57 +5,35 @@
 
 ## How to Build
 
-### Visual Studio
+### Necessary Tools
```

Comment:
I'd say "Requirements" instead of "Necessary Tools". It is what we use in other places.